packages/charts/src/nui-charts.less

```diff
@@ -73,3 +77,151 @@
+
+        &-container {
+            overflow: hidden;
+        }
+
+        &-primary {
+            font-weight: @nui-font-weight-semibold;
+            .setCssVariable(color, nui-color-text-default);
+        }
+
+        &-primary:not(:last-child) {
+            margin-bottom: -@nui-legend-series-content-margin;
+        }
+
+        &-secondary {
+            .setCssVariable(color, nui-color-text-secondary);
+        }
+
+        &-secondary:not(:first-child) {
+            margin-top: -1px;
+        }
+    }
+
+    &--interactive {
+        cursor: pointer;
+
+        & .nui-legend-series__tile-wrapper {
+            position: relative;
+
+            .nui-legend-series__checkbox {
+                &-wrapper {
+                    align-self: center;
+                    position: absolute;
+                    visibility: hidden;
+
+                    top: 50%;
+                    left: 50%;
+                    transform: translate(-50%, -50%);
+                }
+
+                &-mark {
+                    .nui-checkbox__mark();
+                }
+
+                &-glyph {
+                    .nui-checkbox__glyph();
+                    display: none;
+                }
+
+                &--checked {
+                    .nui-legend-series__checkbox-mark {
+                        .setCssVariable(background-color, nui-color-active);
+                        .setCssVariable(border-color, nui-color-active);
+
+                        > .nui-legend-series__checkbox-glyph {
+                            display: block;
+                        }
+                    }
+                }
+            }
+        }
+
+        & .nui-legend-series--deselected,
+        & .nui-legend-series__tile-wrapper:hover {
+            .nui-legend-series__checkbox-wrapper {
+                visibility: visible;
+            }
+
+            .nui-legend-series__tile {
+                visibility: hidden;
+            }
+        }
+
+        & .nui-legend-series__tile-wrapper:hover {
+            .setCssVariable(background-color, nui-color-bg-transparent-hover);
+        }
+
+        &:hover {
+            .setCssVariable(background-color, nui-color-bg-light-hover);
+
+            .nui-legend-series--deselected {
+                .nui-legend-series__tile-wrapper {
+                    .setCssVariable(background-color, nui-color-bg-transparent-hover);
+                }
+            }
+        }
+
+        &:not(:hover) {
+            .nui-legend-series--deselected {
+                .nui-legend-series__tile-wrapper {
+                    .setCssVariable(background-color, nui-color-bg-secondary);
+                }
+            }
+        }
+    }
+
+    &:not(:hover) &--state-hidden {
+        .description-container {
+            opacity: @nui-legend-deemphasized-opacity;
+        }
+    }
+
+    & &--state-deemphasized {
+        opacity: @nui-legend-deemphasized-opacity;
+    }
+}
+
+.description-min-width--unset {
```

Comment:
It seems like this one should go at line 107 just above `.description` since the name isn't really specific enough to be out here on its own. And, once its there, to be safe I wonder if we should add an `!important` next to the rule to ensure that it takes priority. I found the actual usage of this class here:
https://github.com/solarwinds/nova/blob/0c3bd4215aa7054e2488b02de9acf19921bd7bde/packages/dashboards/src/lib/components/proportional-widget/proportional-widget.component.html#L48

Reply:
it will need a `&` in front of it with no space of course